frontend/src/services/authService.js

```
@@ -31,28 +39,38 @@
-  // Check if user is authenticated
-  isAuthenticated: async () => {
+  // Get current user info
+  getCurrentUser: async () => {
     try {
-      const response = await api.get('/auth/status');
-      return response.data.authenticated;
+      const response = await api.get('/auth/me');
+      return response.data.user;
     } catch (error) {
-      return false;
+      throw error;
```

Biome (1.9.4)

[error] The catch clause that only rethrows the original error is useless.

An unnecessary catch clause can be confusing.
Unsafe fix: Remove the try/catch clause.


(lint/complexity/noUselessCatch)